File: 1227-number-of-equivalent-domino-pairs/1227-number-of-equivalent-domino-pairs.c
int numEquivDominoPairs (int** dominoes, int dominoesSize, int* dominoesColSize) {
    int* map = (int*) calloc (sizeof (int), 100);
    int count = 0;
    for (int i = 0; i < dominoesSize; ++i) {
        int u = dominoes [i] [0];
        int v = dominoes [i] [1];
        int num = (u > v) ? v * 10 + u : u * 10 + v;
        count += map [num];
        map [num]++;
    }
    free (map);
    return count;
}

File: 1227-number-of-equivalent-domino-pairs/1227-number-of-equivalent-domino-pairs.py
class Solution:
    def numEquivDominoPairs (self, dominoes : List [List [int]]) -> int:
        hash = [0] * 100
        count = 0
        for d in dominoes:
            u, v = d [0], d [1]
            num = v * 10 + u if u > v else u * 10 + v
            count += hash [num]
            hash [num] += 1
        return count